.eslintrc.yaml

```diff
@@ -52,6 +52,7 @@ rules:
   no-path-concat: off
   no-proto: error
   no-redeclare: error
+  no-restricted-globals: [ error, context ]
```

Comment:
restricting other globals, just complaining about `window.event` and `window.name`, among others...